wdio.conf.js

```diff
@@ -8,4 +8,10 @@ if (process.env.npm_package_name !== 'terra-core') {
   wdioConfig.specs = [path.join(__dirname, 'packages', process.env.npm_package_name, 'tests', 'wdio', '**', '*-spec.js')];
 }
 
+const ci = process.env.CI;
+
+if (ci) {
+  wdioConfig.host = 'localhost';
```

Comment:
This line won't work in the docker dev env.... if you use it and i think it's only me... comment this out to run tests

Reply:
i changed this to look for a different env `travis` this will keep docker dev env working